test/index.js

```diff
@@ -90,6 +107,21 @@ describe('Drip', function(){
         .identify(test.fixture('identify-basic').input)
         .error('bad request status=401 msg=Authentication failed, check your credentials', done);
     });
+
+    it('should retry - no remaining', function(done) {
```

Comment:
Could we add some more tests here? I'm thinking:
- Check that a successful request writes the limits to redis
- Check that when there is no data in redis, the request is made
- Check that when there is data in redis and a non-zero remaining, the request is made